sdk/samples/iot/templates/iothub/test-resources-post.ps1

```diff
@@ -65,12 +65,30 @@ Add-AzIotHubDevice `
 -PrimaryThumbprint $fingerprint `
 -SecondaryThumbprint $fingerprint
 
+if ($LASTEXITCODE -ne 0)
+{
+  Write-Host "Adding cert device failed: LAST_ERROR_CODE=${LAST_ERROR_CODE}"
+  exit $LASTEXITCODE
+}
+
 # Download Baltimore Cert
 curl https://cacerts.digicert.com/BaltimoreCyberTrustRoot.crt.pem > $sourcesDir\BaltimoreCyberTrustRoot.crt.pem
 
+if ($LASTEXITCODE -ne 0)
+{
+  Write-Host "Downloading root cert failed: LAST_ERROR_CODE=${LAST_ERROR_CODE}"
+  exit $LASTEXITCODE
+}
+
 # Link IoTHub to DPS service
 $hubConnectionString = Get-AzIotHubConnectionString -ResourceGroupName $ResourceGroupName -Name $iothubName -KeyName "iothubowner"
 
+if ($LASTEXITCODE -ne 0)
+{
+  Write-Host "Getting connection string for cert device failed: LAST_ERROR_CODE=${LAST_ERROR_CODE}"
```

Comment:
Same here, any non-secret parameter values to the command above might be helpful to log.